app/controllers/rules_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'csv'
```

Comment:
Vitor, can you please add a comment above this 'require' statement so that everyone knows which application feature is using it?

Reply:
Leandro, the inclusion of 'require' is part of a tutorial that i found about export to excel. However, I confess that I didn't test it without 'require' to see if it was still working or not. Anyway, in the lectures we had it was used to read and write in CSV.